### ISSUE TITLE
The silent enable portion is half azz

### ISSUE THREAD

**Issue by rmuhammad4** [MicrosoftDocs/memdocs#255]
There are additional settings in Intune bitlocker that was excluded out. Please edit this document.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 54cccf1d-81fd-5331-aee7-59a35b39cb15
* Version Independent ID: 81d14392-fe74-4fd9-047f-7b8292d84f0c
* Content: [Encrypt Windows 10 devices with BitLocker in Intune - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/protect/encrypt-devices#feedback)
* Content Source: [memdocs/intune/protect/encrypt-devices.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/protect/encrypt-devices.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by Brenduns** [comment]
@rmuhammad4   Information about silently enabling BitLocker remains in content has not been removed. However, hyper-text link to the settings article were removed, as there are now multiple policy types (with the same settings) that can configure this.  Links to settings for both policy and profile types will be restored later today. 

Beyond that, if you can shed light on what you’re not seeing, or would like to see in this content, we can investigate that for an update.

**Comment by rmuhammad4** [comment]
I have been racking my brain why this is not working. I finally had to open a ticket with MS and this is what was sent:

This is to inform you that we have confirmed from our backend team that currently Silent encryption is not supported on hybrid Azure AD joined and it will prompt once while encrypting C drive.

 

We have requested our backend team to update this information in the article and soon you will see this mentioned .

 

I would also like to share some more information about silent encryption hope you find it helpful .

 

More information
BitLocker enforcement on the end device can be of three types:

1.     Automatic (during AADJ for Windows v 1703+) 

2.     Silent (Endpoint protection policy for Windows v 1803+)

3.     Interactive (Endpoint policy for pre Windows v 1803)

·        If your device supports modern Standby (Instant Go) and is HSTI compliant, AADJ will trigger automatic device encryption for Windows version 1703 and above. This does not requires the admin to enforce/deploy an endpoint protection policy.

·        If your device is HSTI compliant but does not supports modern Standby (Instant Go), you would require an endpoint protection policy to enforce silent Bitlocker encryption. Below settings allow for the same.

The OMA-URI reference for the above settings

 

OMA-URI: ./Device/Vendor/MSFT/BitLocker/RequireDeviceEncryption

Value Type: Integer

Value: 1  (1 = Require, 0 = Not Configured)

 

OMA-URI: ./Device/Vendor/MSFT/BitLocker/AllowWarningForOtherDiskEncryption

Value Type: Integer

Note: If the setting "Waiting for other disk encryption" is set to Not configured, then user receives the toast notification and enabling the encryption would require user interaction to go through the Bitlocker activation guide.

 

·        If your device does not supports modern Standby but is HSTI compliant, for pre Windows v 1803, an endpoint protection policy with the above settings will deliver the policy to the device but user will need to manually enable Bitlocker encryption by clicking on the toast notification as received and going through the Bitlocker activation guide.

It is beyond ridiculous how something like this was missed I can imagine there are other miscues. I have been working on this for days the ticket with MS was opened for days and he finally tells me Silent enable is not supported on HAAD clients. why was this not on MS site in the 1st place. I have one unhappy customer along with myself.

**Comment by Brenduns** [comment]
@rmuhammad4   Thank  you for this additional detail.  As of yet, that request hasn't reached our content team. I suspect it might have been passed to the Windows CSP team that owns the BitLocker CSP's involved. Therefore I've also passed this information directly to the Intune development team for review and action. I've also opened a content tracking bug in my own workload queue to track following up with engineering on the correct details for both content and the product UI - and to update this issue when we have more to share, or content revisions on the way.  With this tracked internally, I'll be closing this github issue (which doesn't prevent a future update to it).
